.gitignore

```diff
@@ -30,8 +30,6 @@
 # Ignore master key for decrypting credentials and more.
 /config/master.key
 
-/config/database.yml
```

Comment:
Why do you remove this line from the .gitignore?

Reply:
We need this file in repo, because we can't create files en heroku, and we need to take values from Heroku envs config variables.

Reply:
Fixed! added again to .gitignore